test/functions_linearalgebra.jl

```diff
@@ -41,7 +45,7 @@ function baseline_tests(fact, identity)
             _named isa NamedDimsArray && @testset "Test name for dim $d" for d in 1:ndims(_named)
                 # Don't think it make sense for an factorization to produce properties with
                 # dimension sizes outside 1, 2 or 3
-                @test size(_named, d) in (1, 2, 3)
```

Comment:
I was confused with this test as it looks to be testing the size not the number of dims.